airflow/operators/sensors.py

```diff
@@ -264,7 +340,6 @@ def poke(self, context):
             'Poking for table {self.schema}.{self.table}, '
             'partition {self.partition}'.format(**locals()))
         if not hasattr(self, 'hook'):
-            import airflow.hooks.hive_hooks
```

Comment:
I think because it's not needed due to [this](https://github.com/zodiac/incubator-airflow/blob/a6a86545b758155fb3d18d1684e0ad633e3e5eb8/airflow/operators/sensors.py#L24).